addons/base_setup/views/res_config_settings_views.xml

```diff
@@ -366,6 +366,18 @@
                                 </div>
                             </div>
                         </div>
+
+                        <h2 groups="base.group_no_one">Performances</h2>
+                        <div groups="base.group_no_one" class="row mt16 o_settings_container" name="performances">
```

Comment:
When talking about a program's efficiency, people usually use the term *performance* as an *uncountable*.
```suggestion
                        <h2 groups="base.group_no_one">Performance</h2>
                        <div groups="base.group_no_one" class="row mt16 o_settings_container" name="performance">
```